scr/example-01b_simple-beam-3-span_point-load.py

```diff
@@ -0,0 +1,303 @@
+# PyNite Example
+# Example: Simply supported beam with a point load
+# Units: inches and kips, use consistent units
+
+from operator import ge
+import numpy as np
+from PyNite import FEModel3D
+from PyNite import Reporting
+from PyNite import Visualization
+from pathlib import Path
+import matplotlib.pyplot as plt
+from itertools import accumulate
+
+# Updated parameters for this plot
+plt.style.use("../plt/figstylefile.mplstyle")
+
+
+class Section:
```

Comment:
I've been planning to integrate a "Section" class directly into PyNite for some time. I don't like putting it in the examples as many users may not be very good at object oriented programming. I'd like to keep the examples simple.